### PR TITLE
Finish homepage

### PR DESCRIPTION
**Pull request by vanitabarrett**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -16,6 +17,37 @@ export default function Home() {
           <a>shipping may be delayed</a>
         </Link>
       </div>
+
+      <div className="home-page__info gel-layout">
+        <div className="home-page__info__section gel-layout__item gel-1/3">
+          <img className="home-page__info__image" src="/svgs/sparkles.svg" />
```
Comment (andymsuk) [inline pages/index.tsx]: Need `alt=""` on these images.

File: styles/pages/_home.scss [hunk 2]
```diff
@@ -88,8 +88,74 @@
...
+    @include mq($from: gel-bp-s) {
+      display: block;
+    }
+
+    &__section {
+      display: inline-block;
+      text-align: center;
+      color: $medium-grey;
+      margin: 30px 0;
+
```
Comment (andymsuk) [inline styles/pages/_home.scss]:
```suggestion
```

File: styles/pages/_home.scss [hunk 3]
```diff
@@ -88,8 +88,74 @@
...
+        .home-page__cta__text {
+          color: $white;
+          background-color: $accessible-red;
+        }
+      }
+    }
+
+    &__image {
+      width: 100%;
```
Comment (andymsuk) [inline styles/pages/_home.scss:145]:
```suggestion
      display: block;
      width: 100%;
```
To remove the weird gap below it.

File: pages/index.tsx [hunk 4]
```diff
@@ -16,6 +17,37 @@ export default function Home() {
...
+        </div>
+        <div className="home-page__info__section gel-layout__item gel-1/3">
+          <img className="home-page__info__image" src="/svgs/sustainability.svg" alt=""/>
+          <p className="home-page__info__text">Committed to high quality and sustainable products</p>
+        </div>
+      </div>
+
+      <div className="home-page__cta gel-layout">
+        <a className="home-page__cta__wrapper gel-layout__item gel-1/2@m gel-1/3@l" href="/cards">
+          <img className="home-page__cta__image" src="/featured/all-cards.jpg" />
```
Comment (andymsuk) [inline pages/index.tsx]: Want some `alt` on this and `aria-label` on the links?

File: styles/pages/_home.scss [hunk 5]
```diff
@@ -88,8 +88,74 @@
...
+      &--christmas {
+        object-position: 100% 60%;
+      }
+    }
+
+    &__text {
+      font-size: 1.5rem;
+      background-color: $light-grey;
+      text-align: center;
+      font-family: 'Timothy';
```
Comment (andymsuk) [inline styles/pages/_home.scss]: There's a mixin for this.

File: pages/index.tsx (hunk 4 above)
Comment (vanitabarrett) [reply pages/index.tsx]: Have added blank alt text.

**Review by andymsuk — APPROVED** [review]

**Pull request merged**